step.sh

```diff
@@ -240,7 +240,18 @@ fi
 echo_details "$submit_cmd"
 echo
 
-eval "${submit_cmd}"
+output=$(eval $submit_cmd)
+echo $output
+
+# Adjust the number of `sed -n 2p` if the position of the URL changes in the output
+FIREBASE_CONSOLE_URL=$(echo $output | grep -Eo "(http|https)://[a-zA-Z0-9./?=-_%:-]*" | sed -n 2p)
+echo_info "firebase console url: ${FIREBASE_CONSOLE_URL}"
+envman add --key FIREBASE_CONSOLE_URL --value "${FIREBASE_CONSOLE_URL}"
+
```

Comment:
is there any format of console URLs and distribution URLs?
We should not just rely on the order.

Reply:
@oreno4649 ^^

Reply:
@guness 
As @gremech suggested extract the `release_id_short` is also good enough and put is to env variable.
Is it possible to apply this and release a new version. 

Actually I am waiting for it to integrate to my CI/CD for real project